models/spleen_deepedit_annotation/docs/README.md

```diff
@@ -21,6 +21,34 @@ Input: 3 channels CT image - one channel representing clicks for each segment (i
 
 Output: 2 channels: Label 1: spleen; Label 0: everything else - This depends on the dictionary "label_names" defined in train.json and inference.json ("label_names": {"spleen": 1, "background": 0},)
 
+## Model Performance
+
+The achieved Dice score on the validation set is:
+
+Spleen: 0.971
+
+Command to execute:
+
+```
+python -m monai.bundle run evaluating --meta_file configs/metadata.json --config_file "['configs/train.json','configs/evaluate.json']" --logging_file configs/logging.conf
+```
+
+### Train Dice
+
+![Train Dice](./train_dice_deepedit.png)
+
```

Comment:
`max_epochs` in `train.json` is 500, but the horizontal axis in the figure is 90

Reply:
I didn't train the model for all the 500 epochs. Is that an issue? I just wanted to show the plots I've got during training

Reply:
I'm not sure if it is a hard requirement, but if there is no need to train for 500 epochs, maybe reduce the number in the config file is better (for test and for users to reproduce)?

Hi @Nic-Ma @wyli @tangy5 , what do you think?

Reply:
I can change the default number of epochs. Perhaps for spleen segmentation, it is not needed the 500 epochs. But users may want to use this method for other more complex tasks. Let me know :)

Reply:
It makes sense @diazandr3s , thanks! In this way, maybe we can add a sentence for the figure like "A graph showing the training loss/metric over 90 epochs." (like what we do in the spleen ct segmentation bundle)

Reply:
Makes sense. Just added some comments for this and the Dice score. I think writing a single number doesn't make sense. I commented on that as well.